packages/mdc-ripple/foundation.js

```diff
@@ -325,27 +325,27 @@ class MDCRippleFoundation extends MDCFoundation {
 
     // Avoid reacting to follow-on events fired by touch device after an already-processed user interaction
     const previousActivationEvent = this.previousActivationEvent_;
-    const isSameInteraction = previousActivationEvent && e && previousActivationEvent.type !== e.type;
+    const isSameInteraction = previousActivationEvent && e !== undefined && previousActivationEvent.type !== e.type;
     if (isSameInteraction) {
       return;
     }
 
     activationState.isActivated = true;
-    activationState.isProgrammatic = e === null;
+    activationState.isProgrammatic = e === undefined;
```

Comment:
Come to think of it, would `!e` work here too?

Reply:
Yeah, `!e` would work, but my aim here is to make our typing more strict and obvious.